test/e2e/framework/framework.go

```diff
@@ -266,6 +266,13 @@ func (f *Framework) AfterEach() {
 		LogPodsWithLabels(f.Client, api.NamespaceSystem, ImagePullerLabels)
```

Comment:
I think that was the intent. It was explicity asked for in the original pr, because the prepuller might not finish before we start running tests, and if a test times out image pulling we want to know that we didn't pull that image so we can update the list. I don't have a strong preference eitherway, I don't think we lose anything by being verbose on failure.